tests/unit/proc/test_manager.py

```diff
@@ -0,0 +1,75 @@
+import json
+import os
+import signal
+
+import pytest
+
+from dvc.proc.manager import ProcessManager
+from dvc.proc.process import ProcessInfo
+
+PID_FINISHED = 1234
+PID_RUNNING = 5678
+
+
+def create_process(root: str, name: str, pid: int, returncode=None):
+    info_path = os.path.join(root, name, f"{name}.json")
+    os.makedirs(os.path.join(root, name))
+    process_info = ProcessInfo(
+        pid=pid, stdin=None, stdout=None, stderr=None, returncode=returncode
+    )
+    with open(info_path, "w", encoding="utf-8") as f_w:
+        json.dump(process_info.asdict(), f_w)
```

Comment:
```suggestion
    with open(info_path, "w", encoding="utf-8") as fobj:
        json.dump(process_info.asdict(), fobj)
```

I think we generally just use `fobj` everywhere in the DVC code base, so we should try to stay consistent here